docs/reference/bson.txt

```diff
@@ -14,8 +14,8 @@ Overview
 --------
 
 MongoDB stores data records as BSON documents. BSON is a binary representation
-of :term:`JSON` documents, though it contains more data types than JSON. For the
```

Comment:
Apparently, this term isn't defined and the current docs don't render this in a special way, so I removed it.

Reply:
Thanks. I've seen this error before but never traced it back to this line. I agree this serves no purpose.

It could be something that only builds correctly with the docs team's build tools, since those pull in additional resources used by the server manual (or _did_ if they no longer use Giza).